### DISCUSSION TITLE
How to change the weather entity on the chip card?

### DISCUSSION THREAD

**Discussion by AbhiOommen** [DigiLive/mushroom-strategy#131] [Q&A] [answered]
I have the dashboard installed and working and it is awesome, compared to the amount of effort it would have took me to get it to this level. So THANK YOU to the developer(s).

I am trying to learn how to edit the dashboard? For e.g. is there a simple way to switch the weather entity (I have more than one) on the chip card on the first page. Same question for preventing some entities from showing on the dashboard. I am comfortable using YAML to edit in a code editor.
![mushroom](https://github.com/AalianKhan/mushroom-strategy/assets/11384002/349aa82d-1a82-48cc-92c6-f8452a45e563)

**Comment by AalianKhan** [comment, marked as answer]
Hey,

I would highly suggest going over the [wiki](https://github.com/AalianKhan/mushroom-strategy/wiki) which talks about all the customization options. 

For editing the weather entity, you would use the `weather_entity` option under `chips`. [link to wiki](https://github.com/AalianKhan/mushroom-strategy/wiki/Strategy-Options#chips)

For hiding specific entities, you can follow instructions [here](https://github.com/AalianKhan/mushroom-strategy/wiki/Adding-and-Hiding-entities-and-devices#hiding-specific-entities-and-devices)
